specification/search/data-plane/Azure.Search/preview/2020-06-30/examples/SearchServiceCreateOrUpdateSynonymMap.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "endpoint": "https://myservice.search.windows.net",
+    "synonymMapName": "mysynonymmap",
+    "api-version": "2019-5-06-Preview",
```

Comment:
Wrong version number

Reply:
good catch, updated 2020-Preview version and 2019-Preview versions as well